dev_apps/first-actor/buildsign.md

```diff
@@ -0,0 +1,52 @@
+# Signing your actor
+
+First off, lets build and sign the actor as-is.  This way, we can see if the module is behaving as we expect.
+
+
+<details>
+  <summary>Rust</summary>
+  Build it!
+  `cargo build --release`{{execute}}
+  Sign it!
+  `wash claims sign target/wasm32-unknown-unknown/release/calculator.wasm -c wasmcloud:httpserver --name "calculator" --ver 0.1.0 --rev 0`  
+</details>
+<details>
+  <summary>Go</summary>
+
+  Build it!
+  `mkdir -p build && tinygo build -o build/calculator.wasm -target wasm -no-debug main.go`{{execute}}
+  Sign it!
+  `wash claims sign build/calculator.wasm -c wasmcloud:httpserver --name "calculator" --ver 0.1.0 --rev 0`{{execute}}
```

Comment:
Ditto, use `-q` instead of `-c` here.

Reply:
I would suggest using --http_server for clarity in the intro tutorial